pkg/integration/tests/interactive_rebase/reword_you_are_here_commit_with_editor.go

```diff
@@ -0,0 +1,50 @@
+package interactive_rebase
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var RewordYouAreHereCommitWithEditor = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Rewords the current HEAD commit in an interactive rebase with editor",
+	ExtraCmdArgs: "",
+	Skip:         true,
+	SetupConfig: func(config *config.AppConfig) {
+		// This doesn't work; it still prompts to return after calling git amend,
+		// and this makes the test fail. I couldn't figure out why yet.
+		config.UserConfig.PromptToReturnFromSubprocess = false
+	},
+	SetupRepo: func(shell *Shell) {
+		shell.
+			CreateNCommits(3).
+			SetConfig("core.editor", "sh -c 'echo renamed 02 >.git/COMMIT_EDITMSG'")
```

Comment:
Is this too hacky?

Reply:
I'm fine with it, and it's temporary anyway: at some point we'll need to implement a solution where you can reword any commit from an interactive rebase, by passing lazygit itself (with an environment variable controlling its behaviour) as GIT_EDITOR. That second lazygit process can then talk to the first one and bring up a prompt panel to update the commit message.